big-o-notation/big-o.js

```diff
@@ -0,0 +1,47 @@
+/* exported uniqueQuadratic, uniqueLinear */
+
+function uniqueLinear(words) {
+  const seen = {}; // 1 * 1 = O(1)
+  const unique = []; // 1 * 1 = O(1)
+  for (
+    let i = 0; // 1 * 1 = O(n)
```

Comment:
This only executes once at the beginning of the loop, so it should be O(1).

You got it correct below. In fact, you wrote down the number of times correctly, just missed copied it into the O().